svc/pkg/user-identity/ops/get/src/lib.rs

```diff
@@ -26,15 +41,15 @@ async fn handle(
 		.map(common::Uuid::as_uuid)
 		.collect::<Vec<_>>();
 
-	let identities = ctx
+	let emails = ctx
 		.cache()
 		.fetch_all_proto("user_identity.emails", user_ids.clone(), {
```

Comment:
Needs a new cache key since the proto is changing. I'm just adding "2" to the keys for now, we can find a cleaner system at some point.